plugins/sysinfo_plugin/ufm_sim_web_service/ufm_web_service.py

```diff
@@ -10,44 +10,52 @@
 # provided with the software product.
 #
 
-import asyncio
-import signal
+import logging
+from logging.handlers import RotatingFileHandler
+import os
 
-from ufm_web_sim import UFMWebSim
+from configuration import Configuration
+from base_aiohttp_api import BaseAiohttpServer
+from sysinfo_plugin_api import SysInfoPluginAPI
 
 
-class UFMWebSimProc:
-    """Main class of the UFM web sim daemon
+def create_logger():
     """
+    Create a logger for plugin actions
+    """
+    log_file = Configuration.LOG_FILE_NAME
+    if not os.path.exists(log_file):
+        os.makedirs('/'.join(log_file.split('/')[:-1]), exist_ok=True)
+
+    # pylint: disable=line-too-long
+    logger = logging.getLogger(log_file)
+    format_str = f"%(asctime)-15s ufm-sysinfo-plugin-{log_file} Machine: localhost     %(levelname)-7s: %(message)s"
+    logging.basicConfig(format=format_str, level=Configuration.log_level)
+    rotate_handler = RotatingFileHandler(log_file,maxBytes=Configuration.log_file_max_size, backupCount=Configuration.log_file_backup_count)
+    rotate_handler.setLevel(Configuration.log_level)
+    rotate_handler.setFormatter(logging.Formatter(format_str))
+    logger.addHandler(rotate_handler)
+    return logger
+
+
+def main():
+    """
+    Plugin main function
+    """
+    try:
+        host = "127.0.0.1"
```

Comment:
The host and port should be removed from the main.
They should be configured through the configuration file